grblas/tests/test_scalar.py

```diff
@@ -322,3 +323,37 @@ def test_ndim(s):
     v = Vector.from_values([1], [2])
     assert v.inner(v).ndim == 0
     assert (v @ v).ndim == 0
+
+
+@pytest.mark.skipif("not dtypes._supports_complex")
+# @pytest.mark.parametrize("dtype", ["FC32", "FC64"])  # This segfaults
+@pytest.mark.parametrize("dtype", ["FC64", "FC32"])
```

Comment:
Note that the order we test the dtypes determines whether we segfault or not (on line 337 below).